scss/components/_responsive-embed.scss

```diff
@@ -29,7 +28,10 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 /// Creates a responsive embed container.
 /// @param {List} $ratio [$responsive-embed-ratio] - Ratio to use for the container, formatted as `x by y`.
-@mixin responsive-embed($ratio: $responsive-embed-ratio) {
+@mixin responsive-embed($ratio: null) {
+  @if $ratio == null {
+    $ratio: map-get($responsive-embed-ratios, default);
+  }
```

Comment:
Great idea! I've made this change. Now the `responsive-embed()` mixin accepts a key from the `$responsive-embed-ratios` map or a list formatted as `x by y`.